tests/suite/dfxvm/list.rs

```diff
@@ -87,3 +89,21 @@ fn ignores_non_versions() {
         .success()
         .stdout("0.3.2\n0.7.1\n0.14.2\n0.15.0-beta.1\n");
 }
+
+#[test]
+fn remote_versions() {
+    let home_dir = TempHomeDir::new();
+    let server = ReleaseServer::new(&home_dir);
+
+    server.expect_get_manifest(&manifest_json("0.14.6"));
+
+    home_dir
+        .dfxvm()
+        .arg("list")
+        .arg("--available")
+        .assert()
+        .success()
+        .stderr(is_match("info: fetching http://.*/manifest.json").unwrap())
+        .stdout(contains("0.5.2"))
+        .stdout(contains("0.5.0"));
+}
```

Comment:
Please add a test that passes `--limit 1` and make sure the output only shows the expected version

Reply:
Fixed at https://github.com/dfinity/dfxvm/pull/89/commits/6bf6a71f54c29ded886d079bb1ee2d95b3756d54